### ISSUE TITLE
[Feature-request] Alpine docker image

### ISSUE THREAD

**Issue by danielTobon43** [patrickhoefler/dockerfilegraph#78]
Hi there,

I was wondering if you will build an Alpine docker image since will be smaller in size.

The current image with ubuntu focal is: 
```
ghcr.io/patrickhoefler/dockerfilegraph   latest            0455eb60381c   2 weeks ago         111MB
```

I think with Linux Alpine could be even smaller.

Anyway, thanks for your project. It is awesome.

**Comment by patrickhoefler** [comment]
Hey Daniel,

thanks for your kind words 🙂 

The Alpine-based image is now available (implemented in #79, see the readme and https://github.com/patrickhoefler/dockerfilegraph/pkgs/container/dockerfilegraph for available tags).

Please let me know if it works for you 🙂

**Comment by danielTobon43** [comment]
Wonderful size, thank you very much patrick:
```
ghcr.io/patrickhoefler/dockerfilegraph   0.7.3             fc04afe08bd4   12 hours ago    111MB
ghcr.io/patrickhoefler/dockerfilegraph   0.7.3-alpine      93485687d9ab   12 hours ago    44.9MB
```
